lightly_studio/tests/resolvers/video_frame_resolver/helpers.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from pathlib import Path
+from uuid import UUID
+
+from sqlmodel import Session
+
+from lightly_studio.models.video import VideoCreate, VideoFrameCreate
+from lightly_studio.resolvers import video_frame_resolver, video_resolver
+from tests.resolvers.video_resolver.helpers import VideoStub
+
+
+def create_video_with_frames(
+    session: Session,
+    dataset_id: UUID,
+    video: VideoStub,
+    invert_frame_sorting: bool = False,
+) -> tuple[UUID, list[UUID]]:
+    """Helper function to create a sample."""
```

Comment:
Minor: Let's remove this comment, it does not bring any value. I would rather suggest adding a comment about how the frames are created (using duration and fps)